### ISSUE TITLE
Correct text inserted - challenge status not updated

### ISSUE THREAD

**Issue by Loopaka** [FreeCodeCamp/FreeCodeCamp#2511]
Challenge http://www.freecodecamp.com/challenges/waypoint-label-bootstrap-buttons has an issue. Please describe how to reproduce it, and include links to screenshots if possible.
***************
Using Firefox and Windows 7 --- upon modifying button text, challenge status doesn't get updated, even if the text is correct (see screenshot for details)

********
![cattura](https://cloud.githubusercontent.com/assets/13433879/9425652/e17b8526-4917-11e5-8a23-6d8e0984357d.PNG)

**Comment by issyl0** [comment]
Hi there, @Loopaka! I tried to find you on Gitter, but couldn't. Help here is good too, though. :sunny: 

Try looking a little further up in the editor—you need to add `#target1` to the button with `id=target1`.

Hope that helps!

**Comment by Loopaka** [comment]
Hi Issy, it's the case of a bugged bug-report :) However, the matter remains the same: challenge update is not working (for me, at least), and I just found that is not working also for the jQuery section. Am I the only one reporting on this bug? I have attached a screenshot for that, I can provide other details at will.
![cattura](https://cloud.githubusercontent.com/assets/13433879/9425817/22394f0a-4920-11e5-82af-deb1b6498f77.PNG)
